src/encoded/search_views.py

```diff
@@ -0,0 +1,252 @@
+from pyramid.view import view_config
+
+from snovault.elasticsearch.searches.interfaces import AUDIT_TITLE
+from snovault.elasticsearch.searches.interfaces import MATRIX_TITLE
+from snovault.elasticsearch.searches.interfaces import REPORT_TITLE
+from snovault.elasticsearch.searches.interfaces import SEARCH_TITLE
+from snovault.elasticsearch.searches.interfaces import SUMMARY
+from snovault.elasticsearch.searches.interfaces import SUMMARY_TITLE
+from snovault.elasticsearch.searches.fields import AuditMatrixWithFacetsResponseField
```

Comment:
This is not significant. Why are these (from snovault.elasticsearch.searches.fields ) not group together?